unstructured/file_utils/filetype.py

```diff
@@ -1,14 +1,17 @@
 from __future__ import annotations
 
+import enum
+import functools
 import inspect
 import json
 import os
 import re
 import zipfile
-from enum import Enum
```

Comment:
Any particular reason for switching to `import enum` from `enum import Enum`? I believe the latter is what's suggested in the python documentation and how it's used throughout unstructured

Reply:
Looks like we're currently 50/50 on which way we import enums through the lib. So, we could keep this but we should create a follow-up issue to standardize imports